stampery-trailbot-ubuntu/azuredeploy.json

```diff
@@ -32,7 +32,7 @@
         "description": "Client public key. Please install Trailbot Client in your personal computer, open it and it will generate a key you must paste here."
       }
     },
-    "vmSize": {
+    "size": {
```

Comment:
Oh, now I understand. I'm pushing another commit right now. Thanks a lot for your feedback!